css/style.css

```
@@ -128,6 +142,48 @@ header .toggle {
   height:35px;
 }
 
+.feature {
+  background-color:#f2faff;
+}
+
+.feature .inner {
+  max-width:980px;
+  width:96%;
+  margin:auto;
+}
+
+.feature .wrapper {
+  display:flex;
+  margin-top:40px;
+}
+
+.feature .wrapper .item {
+  width:310px;
+}
+
+.feature .wrapper .item {
+  margin-right:25px;
+}
+
+.feature .wrapper .item + .item +.item {
+  margin:0;
+}
+
+.feature .wrapper .item .img {
+  text-align: center;
+}
+
+.feature .wrapper img {
+  width:198px;
+}
+
+.feature .wrapper .item h3 {
+  font-size:16px;
+  margin:20px 0 30px 0;
+  text-align:center;
+  line-height:16px;
```

高さがフォントサイズと一緒であれば、
```suggestion
  line-height:1;
```
で記載した方がいい気がします。